src/test/java/com/epam/tc/hw6/step/AssertionStep.java

```diff
@@ -0,0 +1,92 @@
+package com.epam.tc.hw6.step;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tc.hw5.cucumber.context.TestContext;
+import com.epam.tc.hw5.cucumber.step.AbstractStep;
```

Comment:
Why `hw5` package is used instead of `hw6`

Reply:
Assertion step removed as it was not in the hw4 is the first place